Expand Diagnose unavailable nodes documentation (#4602)
elastic/docs-content: troubleshoot/monitoring/unavailable-nodes.md

**Resolution**

* [Delete unused data](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-delete).
  
::::{note}
You can delete unused data by running either:
* API calls using the [{{kib}} console](/explore-analyze/query-filter/tools/console.md), if available
* or direct [Elasticsearch API](elasticsearch://reference/elasticsearch/rest-apis/index.md) calls, which you can run when {{es}} has an elected quorum.

> ::::

Reviewer: ```suggestion
::::{note}
You can delete unused data by running either:
* API calls using the [{{kib}} console](/explore-analyze/query-filter/tools/console.md), if available
* direct [{{es}} API](elasticsearch://reference/elasticsearch/rest-apis/index.md) calls, when {{es}} has an elected quorum.
::::
```

**Resolution**

* [Delete unused data](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-delete).
  
::::{note}

> To delete unused data API commands can be ran using Kibana [DevTools](https://www.elastic.co/docs/explore-analyze/query-filter/tools/console) only if it's available, but otherwise API commands can run using the Deployment's [Elasticsearch API Console](https://www.elastic.co/docs/reference/cloud/cloud-hosted/ec-api-console) which is available as long as Elasticsearch has elected quorum.

Reviewer 2: ```suggestion
You can delete unused data by running either:
* the [{{kib}} console](/explore-analyze/query-filter/tools/console.md), if available
* or direct [Elasticsearch API](elasticsearch://reference/elasticsearch/rest-apis/index.md) calls, which you can run when {{es}} has an elected quorum.
```

**Resolution**

* [Delete unused data](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-delete).
  
::::{note}
You can delete unused data by running either:
* API calls using the [{{kib}} console](/explore-analyze/query-filter/tools/console.md), if available
* or direct [Elasticsearch API](elasticsearch://reference/elasticsearch/rest-apis/index.md) calls, which you can run when {{es}} has an elected quorum.

> ::::

Reviewer: ```suggestion
::::{note}
You can delete unused data by running either:
* API calls using the [{{kib}} console](/explore-analyze/query-filter/tools/console.md), if available
* direct [{{es}} API](elasticsearch://reference/elasticsearch/rest-apis/index.md) calls, when {{es}} has an elected quorum.
::::
```

* Disk/memory/CPU saturated:

    * [Delete unused data](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-delete).
      
      ::::{note}
      You can delete unused data by running either:
        * API calls using the [{{kib}} console](/explore-analyze/query-filter/tools/console.md), if available
        * or direct [Elasticsearch API](elasticsearch://reference/elasticsearch/rest-apis/index.md) calls, which you can run when {{es}} has an elected quorum. 

>       ::::

Reviewer: ```suggestion
      ::::{note}
      You can delete unused data by running either:
        * API calls using the [{{kib}} console](/explore-analyze/query-filter/tools/console.md), if available
        * direct [{{es}} API](elasticsearch://reference/elasticsearch/rest-apis/index.md) calls, when {{es}} has an elected quorum. 
      ::::
```
